src/TodoListItem.js

```diff
@@ -1,15 +1,22 @@
 import React from "react";
 
-function TodoListItem({ todo }) {
+function TodoListItem({ todo: { title, id }, onRemoveTodo }) {
   return (
-    <li>
-      <a href={todo.id}>
-        {" "}
-        {todo.title}
-        <span> {todo.id} </span>
-        <span> {todo.lesson} </span>
-      </a>
-    </li>
+    <div>
+      <li>
+        {
+          <a href={id}>
+            {title}
+            <span> {id} </span>
+          </a>
+        }
+        {
+          <button type="button" onClick={() => onRemoveTodo(id)}>
+            Remove
+          </button>
+        }
+      </li>
+    </div>
```

Comment:
Requires changes:
A couple of notes here...
- Div elements are not valid child elements of a `ul`. For example, your current structure is
```html
<ul>
  <div>
    <li> ...etc </li>
   </div>
</ul>
```
Instead, there should be `li` elements...
```html
<ul>
  <li>
     ...etc
  </li>
</ul>
```
- Theses braces here don't do anything since you're just rendering html elements. They can be removed:
```diff
<li>
- {
  <a href={id}>
      {title}
    <span> {id} </span>
  </a>
- }
- {
  <button type="button" onClick={() => onRemoveTodo(id)}>
    Remove
  </button>
- }
</li>
```

Reply:
Thank you Matt for your prompt comment and suggestions. I will correct to the way you suggest.